manifests/art.yaml

```diff
@@ -0,0 +1,15 @@
+updates:
+  - file: "{MAJOR}.{MINOR}/elasticsearch-operator.v{MAJOR}.{MINOR}.0.clusterserviceversion.yaml" # relative to this file
+    update_list:
+    # replace metadata.name value
+    - search: "elasticsearch-operator.v{MAJOR}.{MINOR}.0"
+      replace: "elasticsearch-operator.{FULL_VER}"
+    # replace entire version line, otherwise would replace {MAJOR}.{MINOR}.0 anywhere
+    - search: "version: {MAJOR}.{MINOR}.0"
+      replace: "version: {FULL_VER}"
+    - search: 'olm.skipRange: ">=4.1.0 <{MAJOR}.{MINOR}.0"'
+      replace: 'olm.skipRange: ">=4.1.0 <{FULL_VER}"'
```

Comment:
Should this replacement include `FULL_VER`?  Doesn't the current setting `">=4.1.0 <4.2.0"` say anything in 4.1 not including 4.2.0.  By replacing will we still get the 'latest' zstream... i.e.  4.1.x.  Should the replacement be `>=4.1.0 <={FULL_VER}`

Reply:
@jcantrill 
My understanding is that FULL_VER is the version being built.  Since we don't want it to be considered a valid replacement for itself, we use less-than.